README.md

```diff
@@ -30,16 +30,15 @@ async function logMessagesFromFooBar() {
 
   await bag.open();
 
-  // read all messages from both the '/foo' and '/bar' topics:
-  await bag.readMessages({ topics: ["/foo", "/bar"] }, (result) => {
```

Comment:
Is this a performance pessimization now that we are replacing possible batch calls to the callback with an await between each message?

Reply:
I wrote a benchmark file to test this out by reading our nuscenes file. Turns out it isn't (or at least doesn't factor into the overall performance compared to decompression etc). I will test again with the new generator syntax you recommended.